src/views/account/Homeworks/Document.tsx

```diff
@@ -172,7 +172,7 @@ const HomeworksDocument: Screen<"HomeworksDocument"> = ({ route }) => {
 
           <NativeItem>
             <HTMLView
-              value={`<body>${homework.content}</body>`}
+              value={`<body>${homework.content.replaceAll("<br>", "")}</body>`}
```

Comment:
Peut-être mettre un espace au lieu de rien ?
```suggestion
              value={`<body>${homework.content.replaceAll("<br>", " ")}</body>`}
```